library/protobuf-es/v0.1.0/buf.plugin.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/library/protobuf-es
+plugin_version: v0.1.0
+source_url: https://github.com/bufbuild/protobuf-es
+registry:
+  npm:
+    deps:
+      - package: '@bufbuild/protobuf'
+        version: ^0.0.10
```

Comment:
This version needs to be bumped. 

Another observation:

1. writing the YAML file back out strips the quotes and/or converts them to 
2. Removes the comments

```diff
< plugin_version: v0.0.10
---
> plugin_version: v0.1.0
8,10c8,9
<       # https://www.npmjs.com/package/@bufbuild/protobuf
<       - package: "@bufbuild/protobuf"
<         version: "^0.0.10"
---
>       - package: '@bufbuild/protobuf'
>         version: ^0.0.10
```

I wonder if the regex approach previously was good enough for our purposes.

Reply:
Manually made these changes match the previous version and just bumped version number.